docs/UserGuide.md

```diff
@@ -189,4 +189,11 @@ Action | Format, Examples
 **Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **List** | `list`
+**Group** | `group GROUP_NAME` <br> e.g., `group CS2103 Project Group`
+**Groups** | `groups`
+**Delete Group** | `deleteG g/GROUP_INDEX`<br> e.g., `deleteG g/2`
+**Adding people to a group** | `addG PERSON_INDEX_1, PERSON_INDEX_2.. g/GROUP_INDE`<br> e.g.,`addG 1, 2, 3 g/2`
+**Join** | `join PERSON_INDEX g/GROUP_INDEX_1[,GROUP_INDEX_2]...`<br> e.g., `join 2 g/1,3,6,8`
+**Mates** | `mates GROUP_INDEX`<br> e.g., `mates 7`
```

Comment:
I think you missed out the command that deletes a group and everybody within it. Then again, the command name, currently `delete`, clashes with the other current `delete` used for deleting people. Maybe this should be brought up to the rest of the team.

Reply:
Fixed with @asherhy